src/App.tsx

```diff
@@ -186,37 +187,31 @@ export default function App() {
             .join('')};
         </div> */}
       {choices?.length > 0 && (
-        <div
-          ref={scrollRef}
+        <SimpleBar
+          scrollableNodeProps={{ ref: scrollRef }}
+          className="px-4 pb-4 flex flex-col text-black dark:text-white w-full max-h-screen focus:border-none focus:outline-none outline-none"
           // style={{ maxHeight: '85vh' }}
-          className="flex flex-col text-black   dark:text-white overflow-y-scroll overflow-x-hidden max-h-screen focus:border-none focus:outline-none outline-none"
         >
           {((choices as any[]) || []).map((choice, i) => (
             // eslint-disable-next-line jsx-a11y/click-events-have-key-events
             <button
               type="button"
               key={choice.uuid}
-              style={{ minHeight: '4rem' }}
               className={`
+              w-full
+              my-1
               h-16
-
-              hover:bg-gray-400
-              dark:hover:bg-black
-              dark:hover:bg-opacity-100
-              placeholder-gray-700
-              dark:placeholder-gray-300
+              dark:hover:bg-gray-800
+              hover:bg-gray-100
               whitespace-nowrap
               text-left
               flex
               flex-col
               text-xl
               pl-4
+              rounded-lg
               justify-center
-              ${
-                index === i
-                  ? `dark:bg-black bg-white shadow-large shadow-inner border-black dark:border-white border-b-2 border-t-2 border-opacity-70`
-                  : ``
-              }`}
+              ${index === i ? `dark:bg-gray-800 bg-gray-100` : ``}`}
```

Comment:
I removed the border since I used background color with enough contrast to distinguish selected items.